### PR TITLE
add macOS Disable System Updates TTP

### PR DESCRIPTION
**Pull request by VVX7**
# Proposed Changes

macOS Disable System Updates TTP + readme

I didn't see an example of a TTP that requires admin permissions so I've added one. Currently this TTP assumes the user is a member of sudoers and prompts for their password at run time. Wondering if you had plans to handle this differently.

## Related Issue(s)

n/a

## Testing

Executed on local machine to confirm effects and cleanup.

## Documentation

Included in README.md

## Screenshots/GIFs (optional)

n/a

## Checklist

- [ x ] Ran `mage runprecommit` locally and fixed any issues that arose.
- [ x ] Curated your commit(s) so they are legible and easy to read and understand.
- [ x ] 🚀

### CODE REVIEW

**Comment by VVX7** [comment]
huh local precommit didn't throw an error on the trailing whitespace. gonna write a few more ttps just to go through the motions. will pull a fresh copy to see if it happens again.

**Comment by l50** [comment]
> huh local precommit didn't throw an error on the trailing whitespace. gonna write a few more ttps just to go through the motions. will pull a fresh copy to see if it happens again.

That's odd. What version of pre-commit are you running? 

We also provide a [dev container](https://github.com/facebookincubator/TTPForge/tree/main/.devcontainer/bash), which will be quite representative of what is happening in our github actions.

**Review by l50 — CHANGES_REQUESTED** [review]
Let's make sure you get credit in badge form for your awesome submission :)

Also, if you have any other info around the pre-commit issue you ran into (see comment for more info), please LMK so I can try and replicate.

**Comment by l50** [comment]
This is so awesome, thank you so much for another TTP!!\

File: ttps/defense-evasion/macos/disable-system-updates/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# macOS Disable System Updates
+
+![Meta TTP](https://img.shields.io/badge/Meta_TTP-blue)
```
[the remark this answers is not in the document]
Comment (VVX7) [reply ttps/defense-evasion/macos/disable-system-updates/README.md:3]: Yep! Good idea to differentiate community TTPs from those authored by your team.

**Review by l50 — APPROVED** [review]

**Pull request merged**